### PR TITLE
CSS fixes

### PR DESCRIPTION
**Pull request by seanyesmunt**
https://github.com/lbryio/lbry-app/issues/1503
https://github.com/lbryio/lbry-app/issues/1437
https://github.com/lbryio/lbry-app/issues/1592

Also added minor css fixes accross the app.

### CODE REVIEW

**Comment by seanyesmunt** [comment]
@kauffj can you give this a review?

File: package.json [hunk 1]
```diff
@@ -126,7 +126,7 @@
     "yarn": "^1.3"
   },
   "lbrySettings": {
-    "lbrynetDaemonVersion": "0.20.0rc14",
+    "lbrynetDaemonVersion": "0.20.0",
```
Comment (kauffj) [inline package.json]: nice CSS fix :grin:

File: src/renderer/scss/_gui.scss [hunk 2]
```diff
@@ -121,6 +121,10 @@ input::placeholder {
   opacity: 0.5;
 }
 
+label {
+  cursor: pointer;
```
Comment (kauffj) [inline src/renderer/scss/_gui.scss]: I'm guessing this ought to be `label[for]`, if not something even stronger or more restrictive.

File: src/renderer/page/file/view.jsx [hunk 3]
```diff
@@ -184,21 +192,23 @@ class FilePage extends React.Component<Props> {
...
-                      <SubscribeButton uri={subscriptionUri} channelName={channelName} />
-                    </React.Fragment>
-                  )}
-                  {speechSharable && (
-                    <ViewOnWebButton claimId={claim.claim_id} claimName={claim.name} />
+                    <SubscribeButton uri={subscriptionUri} channelName={channelName} />
                   )}
                 </div>
               </div>
+              <div className="card__actions card__actions--end">
```
Comment (kauffj) [inline src/renderer/page/file/view.jsx]: Should this be rendered if it is empty?

File: src/renderer/scss/component/_content.scss [hunk 4]
```diff
@@ -78,6 +78,20 @@
...
+  background-color: black;
+  width: 100%;
+  padding-top: var(--video-aspect-ratio);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--color-white);
+
+  &.content__empty--nsfw {
+    background-color: var(--color-error);
```
Comment (kauffj) [inline src/renderer/scss/component/_content.scss]: Error coloring should be reserved for errors exclusively.

File: src/renderer/page/file/view.jsx [hunk 5]
```diff
@@ -115,7 +116,7 @@ class FilePage extends React.Component<Props> {
     const { height, channel_name: channelName, value } = claim;
     const mediaType = Lbry.getMediaType(contentType);
     const isPlayable =
-      Object.values(player.mime).indexOf(contentType) !== -1 || mediaType === 'audio';
+      Object.values(player.mime).indexOf(mediaType) !== -1 || mediaType === 'audio';
```
Comment (kauffj) [inline src/renderer/page/file/view.jsx]: `Array.includes()` is a slightly superior way of doing this now

File: src/renderer/component/walletSend/view.jsx [hunk 6]
```diff
@@ -56,8 +56,10 @@ class WalletSend extends React.PureComponent<Props> {
                     label={__('Amount')}
                     postfix={__('LBC')}
                     className="input--price-amount"
+                    fixClass="form-field--fix-no-height"
                     min="0"
                     step="any"
+                    placeholder="5"
```
Comment (kauffj) [inline src/renderer/component/walletSend/view.jsx]: Sometimes placeholders can cause confusion among users who think the placeholder values are actually in the input.

While styling is the primary way to handle this, placeholder values that are less likely to be real values can also help.

(E.g. `12.34` may be a better cue than `5`)

File: src/renderer/component/common/form-components/form-field.jsx [hunk 7]
```diff
@@ -20,6 +20,7 @@ type Props = {
   placeholder?: string | number,
   children?: React.Node,
   stretch?: boolean,
+  fixClass?: string, // class applied to prefix/postfix label
```
Comment (kauffj) [inline src/renderer/component/common/form-components/form-field.jsx]: `affixClass` would be a better term

**Pull request merged**